flixel/FlxSprite.hx

```diff
@@ -983,13 +983,15 @@ class FlxSprite extends FlxObject
 		
 		var minX:Float = x - offset.x - Camera.scroll.x * scrollFactor.x;
 		var minY:Float = y - offset.y - Camera.scroll.y * scrollFactor.y;
+		var zoomOffsetX:Float = Camera.zoomOffsetX;
+		var zoomOffsetY:Float = Camera.zoomOffsetY;
 		
 		if ((angle == 0 || bakedRotationAngle > 0) && (scale.x == 1) && (scale.y == 1))
 		{
-			if (minX > Camera.width || minX + frameWidth < 0)
+			if (minX > Camera.width + zoomOffsetX || minX + frameWidth < -zoomOffsetX)
```

Comment:
maybe it's better to add new property to the camera (`viewport:FlxRect` for example) which will hold these values:
`-zoomOffsetX`, `Camera.width + zoomOffsetX`, `-zoomOffsetY` and `Camera.height + zoomOffsetY`), so no need to recalculate these values too